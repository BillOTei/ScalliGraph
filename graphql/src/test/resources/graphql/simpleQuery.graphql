query AllPerson {
    allPeople {
        name {
            sort {
                toList
            }
        }
    }
}