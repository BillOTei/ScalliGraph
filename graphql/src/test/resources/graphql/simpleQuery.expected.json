{
  "data": {
    "allPeople": {
      "name": {
        "sort": {
          "toList": [
            "franck",
            "josh",
            "marc",
            "marko",
            "peter",
            "vadas"
          ]
        }
      }
    }
  }
}